src/Workspaces/Core/Portable/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -818,7 +818,7 @@ private string GetWhitespaceString(LineColumn lineColumn, LineColumnDelta delta)
             var useTabOnlyForIndentation = this.OptionSet.GetOption(FormattingOptions.UseTabOnlyForIndentation, this.Language);
```

Comment:
With this change the (completely untested :frowning:) UseTabOnlyForIndentation option has no effect anymore.  The idea of that option is to use Tabs for the base indentation of the statement, and then spaces for alignment of consecutive lines after that initial set of tabs.

Perhaps we should either:

1. Remove the option, since it's untested and not exposed through the UI.
2. Add it back, put some tests in place, and use a different method to calculate alignment spaces and the spaces between tokens.